### ISSUE TITLE
The error occur when file name contain (+) character

### ISSUE THREAD

**Issue by wongchipan** [coomar2841/android-multipicker-library#13]
The error occur when file name contain (+) character after the selection of file

The original file name is Capture+_2016-03.png

Unable to decode stream: java.io.FileNotFoundException: /
storage/emulated/0/TestApp/TestApp-Pictures/Capture _2016-03.png: open failed: ENOENT (No such file or directory)

**Comment by coomar2841** [comment]
Which version you have? Can't seem to reproduce this problem. Also, if you can share Device info and version, that would be helpful.

**Comment by wongchipan** [comment]
My device is lg g4 (android 6.0) and library version is 0.9.8

Log File
[log.txt](https://github.com/coomar2841/android-multipicker-library/files/198885/log.txt)
 
Code

```
ImagePicker imagePicker = new ImagePicker((Activity) context);

 public void open() {

        imagePicker.setImagePickerCallback(

                new ImagePickerCallback() {

                    @Override
                    public void onImagesChosen(List<ChosenImage> images) {

                        for (int i = 0; i < images.size(); i++) {
                             // original filename is Capture+_2016-04-01-10-25-47.png
                             // getOriginalPath() value is not contain (+) character 
                             // Capture _2016-04-01-10-25-47.png'
                            String file = images.get(i).getOriginalPath();
                            Toast.makeText(context, file + " ", Toast.LENGTH_SHORT).show();
                        }

                    }
                    @Override
                    public void onError(String message) {

                    }
                }
        );
        imagePicker.shouldGenerateMetadata(false);
        imagePicker.shouldGenerateThumbnails(false);
        imagePicker.pickImage();
    }
```


 ```
@Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        if(resultCode == RESULT_OK) {
            if(requestCode == Picker.PICK_IMAGE_DEVICE) {
                imagePicker.submit(data);
            }
        }
    }
```

**Comment by coomar2841** [comment]
Ok. Will look at it. Ca you try the sample app and see if that is also failing? Just to be sure. Just choose the same image.

**Comment by coomar2841** [comment]
Can you try this version? `0.9.9-beta`

**Comment by wongchipan** [comment]
The problem has been solved, thank you

**Comment by wongchipan** [comment]
The problem has been solved in the 0.99-beta version, thank you
